taskw/warrior.py

```diff
@@ -32,9 +33,36 @@
 logger = logging.getLogger(__name__)
 
 
-# Location of configuration file: either specified by TASKRC environment
-# variable, or ~/.taskrc (default).
-TASKRC = os.getenv("TASKRC", "~/.taskrc")
+def find_taskrc():
+    """
+    Find the location of the taskwarrior configuration file.
+
+    First checks ${TASKRC} environment variable, then falls back to ~/.taskrc and finally ${XDG_CONFIG_DIR}/task/taskrc.
+
+    Raises FileNotFoundErrors if either
+      * Specified taskrc is not a file
+      * No taskrc was found
+    """
+    if "TASKRC" in os.environ.keys():
```

Comment:
FYI, python automatically applies the `in` operator to dictionary keys so would be idiomatic to write this as `if "TASKRC" in os.environ:`